R/backend-tuneR.R

```diff
@@ -0,0 +1,34 @@
+
+#' tuneR_loader
+#'
+#' @keywords internal
+tuneR_loader <- function(
+    filepath,
+    offset = 0L,
+    duration = Inf,
+    unit = "samples") {
+  package_required("tuneR")
```

Comment:
nit: We could change that to `rlang::check_installed()` which asks the user to install the package on interactive sessions.

Reply:
will do!